Configuration/Geometry/python/GeometryExtended2016Reco_cff.py

```diff
@@ -30,4 +30,5 @@
 from Geometry.CaloEventSetup.EcalTrigTowerConstituents_cfi import *
 from Geometry.EcalMapping.EcalMapping_cfi import *
 from Geometry.EcalMapping.EcalMappingRecord_cfi import *
+from Geometry.HcalCommonData.hcalDDConstants_cff import *
```

Comment:
ok, it's in bsunanda/Run2-hcx22, but the definition in it is wrong for DB: hcalParameters_cfi should not be included in this case.